app/src/main/res/layout/display_bottom_sheet.xml

```diff
@@ -55,6 +55,45 @@
                     android:text="@string/comfortable_grid" />
             </RadioGroup>
 
+            <TextView
+                android:id="@+id/textView"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="16dp"
+                android:layout_marginEnd="12dp"
+                android:text="@string/edit_text_show_hide_scrollbar"
+                android:textAppearance="@style/TextAppearance.Medium.Body2" />
```

Comment:
This text view should have same and layout stuff as the other radio group

   <com.google.android.material.textview.MaterialTextView
                style="@style/TextAppearance.MaterialComponents.Headline6"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:paddingStart="16dp"
                android:paddingEnd="12dp"
                android:text= your new string />